src/encoded/tests/test_upgrade_analyte.py

```diff
@@ -0,0 +1,31 @@
+from typing import Any, Dict
+
+import pytest
+from pyramid.router import Router
+
+from .utils import get_upgrader
+
+
+@pytest.mark.parametrize(
+    "analyte,expected",
+    [
+        ({}, {"schema_version": "2"}),
+        (
+            {"components": [], "schema_version": "1"},
+            {"schema_version": "2"},
```

Comment:
This test is impossible no? The previous schema would throw a validation error if you had this.

Reply:
Yup, goes back to the being safer than I need to be here; don't know why I did so. AFAIK, we don't have the means to test upgraders on items in the database properly (i.e. as it occurs on production), so we don't actually validate the test data. Would require validating against two different versions of the schema.